.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,123 @@
+        -    [] Using WooRelease, perform the live release with e.g., `php woorelease.php wporg:release --product_version=9.9.0-rc.1 --zip_file=ZIP_FILE_PATH --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/release/9.9 --release`
+        -    [] If the release command reports SVN-related timeouts, validate versions on https://wordpress.org/plugins/woocommerce/advanced/ and https://plugins.svn.wordpress.org/woocommerce/tags/ pages and continue if those are correct
+        -    [] Upon receiving email `[WordPress Plugin Directory] Pending release for WooCommerce`, confirm the release
+        -    [] Verify the new version is listed and downloadable on https://wordpress.org/plugins/woocommerce/advanced/ page
+
+        #### Stage and bump stable tag
+
+        -    [] Manually create a release entry on https://github.com/woocommerce/woocommerce/releases (use the version as tag, upload the generated zip, and pick the proper release branch)
+        -    [] Verify new PRs created by `github-actions` and merge ones related to the release
+
+        #### Send comms out
+
+        -    [] Coordinate with the Developer Advocacy team to update the pre-release post on developer.woocommerce.com
+        -    [] Notify #woo-core-releases and #woo-announcements Slack channels of the release with a link to the release post
+        -    [] Post an update in the internal release P2 (see pf59Ax-2Nt-p2 for reference)
+
+        #### Post-release monitoring
+
+        -    [] Follow post-release monitoring guidelines
+
+  - type: markdown
+    attributes:
+      value: |
+        ### Final release
+        
+        #### Get release artifact ready
+        
+        -    [] Validate that the stable tag in `readme.txt` in the release branch matches the stable tag in `readme.txt` in the SVN `trunk` ([here](https://plugins.svn.wordpress.org/woocommerce/trunk/readme.txt))
+        -    [] Generate [changelog](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt) ([workflow](https://github.com/woocommerce/woocommerce/actions/workflows/new-cfe-prr-pre-build-compile-changelog-file.yml))
+        -    [] Create a PR to bump version in source code with `GITHUB_TOKEN=<your_PAT_token> pnpm utils code-freeze version-bump -b release/9.9 9.9.0`
+        -    [] Review (ensure PR CI-checks are passing) and merge the PR (ensure merge CI-checks are passing)
+        -    [] Build zip file ([workflow](https://github.com/woocommerce/woocommerce/actions/workflows/build-release-zip-file.yml))
+        -    [] Validate version and stable tag in 'woocommerce.php' and `readme.txt` for the generated zip file
+
+        #### Release to WordPress.org
+
+        -    [] Notify Atomic that a release is about to be made (via P2 and Slack, see  p9o2xV-5y1-p2 and p1747039405416989-slack-C7YPW6K40 for reference) before deploying release
```

Comment:
This has been historically done well in advance to give Atomic time to respond and also because they usually do updates at a certain time during the day (I believe ~12 UTC). Given this is a good time to simplify, I think we should confirm with them whether there is indeed a time requirement (and if so, what it is exactly so we can include it here) and whether one of the 2 pings (P2 or Slack) would be enough.

IMO, Slack is slightly more useful as we can discuss things "live" with them but in any case reducing duplication would be good.

Reply:
> Should we move this item under Get release artifact ready or Release to WordPress.org?

I think this can be part of `Release to WordPress.org`, though maybe we can have a section for all things Atomic (including notifying, monitoring, etc.)? We can always iterate on the format, so feel free to add to wherever section you think it fits best.


> I like the idea to simplify and use Slack for comms, can we confirm with folks from Atomic if they are good with it?

I've asked about this in p1748357764216009-slack-C7YPW6K40.